### PR TITLE
:bug: Source Facebook marketing: reduce start date by one day

### PR DESCRIPTION
**Pull request by davydov-d**
## What
https://github.com/airbytehq/oncall/issues/1975

## How
According to the logs, the requested date isn't actually older than 37 months. But the exception says it is. The only reasonable explanation to me of what's going on is possible timezone difference causing the start date be out of 37 months bound. I think we need to increase the start date by one day in order to stay within the appropriate date-time boundaries.

## 🚨 User Impact 🚨
 - Less data (max. difference - one day) **may** be replicated in case the start date in user's config is set to more then 37 months ago

### CODE REVIEW

**Comment by octavia-squidington-iii** [comment]
##  source-facebook-marketing test report (commit [`efbff4d9dd`](https://github.com/airbytehq/airbyte/pull/27483/commits/efbff4d9ddaa1c7987136a6f59a643b2cffc4be3)) - ❌

⏲️  Total pipeline duration: 4029 seconds

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml | ✅        |
| Connector version semver check.                                                  | ✅        |
| Connector version increment check.                                               | ✅        |
| QA checks                                                                        | ✅        |
| Code format checks                                                               | ❌        |
| Connector package install                                                        | ✅        |
| Build source-facebook-marketing docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                       | ✅        |
| Integration tests                                                                | ✅        |
| Acceptance tests                                                                 | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5315380814)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-facebook-marketing test
```

**Comment by octavia-squidington-iii** [comment]
##  source-facebook-marketing test report (commit [`f830738807`](https://github.com/airbytehq/airbyte/pull/27483/commits/f8307388073c8fa694b73c63842d01f179c39a0c)) - ❌

⏲️  Total pipeline duration: 3802 seconds

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml | ✅        |
| Connector version semver check.                                                  | ✅        |
| Connector version increment check.                                               | ✅        |
| QA checks                                                                        | ✅        |
| Code format checks                                                               | ❌        |
| Connector package install                                                        | ✅        |
| Build source-facebook-marketing docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                       | ✅        |
| Integration tests                                                                | ✅        |
| Acceptance tests                                                                 | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5315386911)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-facebook-marketing test
```

File: airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ data:
   registries:
     cloud:
       enabled: true
-      dockerImageTag: 0.4.2 
+      dockerImageTag: 0.4.3
```
Comment (bnchrch) [inline airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml:17]: You should be able to remove this override now that its the same as the default

**Review by bnchrch — APPROVED** [review]

**Comment by octavia-squidington-iii** [comment]
##  source-facebook-marketing test report (commit [`9d00b0fc8b`](https://github.com/airbytehq/airbyte/pull/27483/commits/9d00b0fc8b70d1b0d377b5a5f258c22a9c7fd796)) - ❌

⏲️  Total pipeline duration: 3628 seconds

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml | ✅        |
| Connector version semver check.                                                  | ✅        |
| Connector version increment check.                                               | ✅        |
| QA checks                                                                        | ❌        |
| Code format checks                                                               | ✅        |
| Connector package install                                                        | ✅        |
| Build source-facebook-marketing docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                       | ✅        |
| Integration tests                                                                | ✅        |
| Acceptance tests                                                                 | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5330918805)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-facebook-marketing test
```

**Comment by octavia-squidington-iii** [comment]
##  source-facebook-marketing test report (commit [`6c7688a673`](https://github.com/airbytehq/airbyte/pull/27483/commits/6c7688a6732ab4efabf6191a2ee4700f963a0cbd)) - ✅

⏲️  Total pipeline duration: 3162 seconds

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml | ✅        |
| Connector version semver check.                                                  | ✅        |
| Connector version increment check.                                               | ✅        |
| QA checks                                                                        | ✅        |
| Code format checks                                                               | ✅        |
| Connector package install                                                        | ✅        |
| Build source-facebook-marketing docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                       | ✅        |
| Integration tests                                                                | ✅        |
| Acceptance tests                                                                 | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5331412454)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-facebook-marketing test
```

**Comment by octavia-squidington-iii** [comment]
##  source-facebook-marketing test report (commit [`3f1477d9a9`](https://github.com/airbytehq/airbyte/pull/27483/commits/3f1477d9a9f2aaaba6740c7e2de2080958ce998c)) - ❌

⏲️  Total pipeline duration: 3359 seconds

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml | ✅        |
| Connector version semver check.                                                  | ✅        |
| Connector version increment check.                                               | ✅        |
| QA checks                                                                        | ✅        |
| Code format checks                                                               | ✅        |
| Connector package install                                                        | ✅        |
| Build source-facebook-marketing docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                       | ✅        |
| Integration tests                                                                | ✅        |
| Acceptance tests                                                                 | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5332809796)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-facebook-marketing test
```

**Comment by octavia-squidington-iii** [comment]
##  source-facebook-marketing test report (commit [`0e0add5f2f`](https://github.com/airbytehq/airbyte/pull/27483/commits/0e0add5f2f66571f58a1d33e1d0822662107eabc)) - ❌

⏲️  Total pipeline duration: 3968 seconds

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml | ✅        |
| Connector version semver check.                                                  | ✅        |
| Connector version increment check.                                               | ✅        |
| QA checks                                                                        | ✅        |
| Code format checks                                                               | ✅        |
| Connector package install                                                        | ✅        |
| Build source-facebook-marketing docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                       | ✅        |
| Integration tests                                                                | ✅        |
| Acceptance tests                                                                 | ❌        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5333678236)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-facebook-marketing test
```

**Comment by octavia-squidington-iii** [comment]
##  source-facebook-marketing test report (commit [`d86437bafb`](https://github.com/airbytehq/airbyte/pull/27483/commits/d86437bafbbee97bc0b4059cb0545c662b8268d3)) - ✅

⏲️  Total pipeline duration: 3780 seconds

| Step                                                                             | Result   |
|:---------------------------------------------------------------------------------|:---------|
| Validate airbyte-integrations/connectors/source-facebook-marketing/metadata.yaml | ✅        |
| Connector version semver check.                                                  | ✅        |
| Connector version increment check.                                               | ✅        |
| QA checks                                                                        | ✅        |
| Code format checks                                                               | ✅        |
| Connector package install                                                        | ✅        |
| Build source-facebook-marketing docker image for platform linux/x86_64           | ✅        |
| Unit tests                                                                       | ✅        |
| Integration tests                                                                | ✅        |
| Acceptance tests                                                                 | ✅        |

🔗 [View the logs here](https://github.com/airbytehq/airbyte/actions/runs/5335301332)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/tools/ci_connector_ops/ci_connector_ops/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=source-facebook-marketing test
```

**Pull request merged**